developer_manual/design/popovermenu.rst

```diff
@@ -0,0 +1,68 @@
+.. sectionauthor:: John Molakvoæ <skjnldsv@protonmail.com>
+.. codeauthor:: John Molakvoæ <skjnldsv@protonmail.com>
+..  _popovermenu:
+
+===============
+Popover Menu
+===============
+
+What is a popover menu
+=======================
+
+This is a quick menu that open on click. We're usually using the three dots icons on nextcloud.
+
+This is exactly the same as the :ref:`navigation menu <navigation_menu>`. The only difference is the popovermenu class.
+
+.. image:: ../images/popovermenu.png
+
+Basic layout
+=============
+
+.. code-block:: html
+
+	<div class="popovermenu">
+	    <ul>
+	        <li>
+	            <a href="#" class="icon-details">
+	                <span>Details</span>
+	            </a>
+	        </li>
+	        <li>
+	            <button class="icon-details">
+	                <span>Details</span>
+	            </button>
+	        </li>
+	        <li>
+	            <button>
+	                <span class="icon-details"></span><span>Details</span>
+	            </button>
+	        </li>
+	        <li>
+	            <a>
+	                <span class="icon-details"></span><span>Details</span>
+	            </a>
+	        </li>
+	    <ul>
+	</div>
+
+Technical details
+==================
+
+* The only allowed menu items elements are **a** and **button**.
+* You can mix between a and button on the same menu (in case of form or direct link) like the example above
+* You need to put the entire menu just after the three dot icon ``<div><span class="icon-more"></span><div class="popovermenu">...</div></div>``
+* You do not need js, css only is ok for positioning. Js is **still** required to handle the hide/show.
```

Comment:
Let’s always capitalize JS and CSS as it is done, also so it stands out in the text.